data/example/3-sample-tags.tsv

```diff
@@ -0,0 +1,92 @@
+Document,Type,ID,Offset,End

```

Comment:
File outdated. Got to rerun analysis.

Perhaps, let's add a `README.md` to `data/example` which lists the shell commands to run these examples.